### ISSUE TITLE
What is the unit of rendered depth from splatfacto?!

### ISSUE THREAD

**Issue by 7yzx** [nerfstudio-project/nerfstudio#3622]
I very confused about depth unit , I just want to add depth loss for splatfactor, I visualize the rendered depth(RED),and my gt depth from apple lidar(GREEN) at same place, it seems to multiply by 10 to match the lidar depth
From RGB image 1.2m is right.
![Image](https://github.com/user-attachments/assets/f2e60b0c-42b5-4f62-ac3e-c71c44759301)

this is my args :

```json
      "args": [
        "splatfacto-big",
        "--data","D:/yzx_code/test_data/dianxingshuzishenghuoBackend/282/282_studio" ,
        "--output-dir", "D:/yzx_code/output_result/282/splatfactor_b_c_mcmc_depth" ,
        "--pipeline.model.use_bilateral_grid", "True",
        "--pipeline.model.output-depth-during-training", "True",
        "--pipeline.model.strategy", "mcmc",
        "--pipeline.model.vis_depth", "D:/yzx_code/test_data/dianxingshuzishenghuoBackend/282/vis_original_nerfstudio_no_depth",
        "--vis", "wandb",
        "colmap",
      ]
``` 

My visualize code is add in `nerfstudio\models\splatfacto.py` line 716
```python
        if len(self.config.vis_depth) >0:
            if not os.path.exists(self.config.vis_depth):
                os.makedirs(self.config.vis_depth)
            vis_gt_image = gt_img.clone()
            vis_gt_image = vis_gt_image.permute(2,0,1).unsqueeze(0)
            # gt_depth = batch["depth"].unsqueeze(0).unsqueeze(0).cuda()

            pred_depth = outputs["depth"].permute(2,0,1).unsqueeze(0)
            # pred_resized = torch.nn.functional.interpolate(pred_depth, size=(gt_depth.shape[2], gt_depth.shape[3]), mode="bilinear", align_corners=False)

            # gt_conf = batch["depth_conf"].unsqueeze(0).unsqueeze(0).cuda()
            if not os.path.exists(self.config.vis_depth):
                os.makedirs(self.config.vis_depth)
            # self.save_depth_tensor_to_tiff(gt_depth, os.path.join(self.config.vis_depth,"{}-gt.tiff".format(batch["image_idx"])))
            self.save_depth_tensor_to_tiff(pred_depth, os.path.join(self.config.vis_depth,"{}-rendered.tiff".format(batch["image_idx"])))
            # self.save_depth_tensor_to_tiff(gt_conf, os.path.join(self.config.vis_depth,"{}-confidence_mask.tiff".format(batch["image_idx"])))
            self.save_depth_tensor_to_tiff(vis_gt_image, os.path.join(self.config.vis_depth,"{}-gt_image.tiff".format(batch["image_idx"])))
```
```python
    def save_depth_tensor_to_tiff(self, tensor, output_path):
        """
        将PyTorch tensor保存为TIFF格式的深度图
        Args:
            tensor: PyTorch tensor (深度图数据，通常是[H, W]或[1, H, W]的float类型)
            output_path: 保存路径，例如 'depth_map.tiff'
        """
        # 确保tensor是CPU上的
        if tensor.is_cuda:
            tensor = tensor.cpu()
        
        # 转换为numpy数组
        if len(tensor.shape) == 3:  # 如果是[1, H, W]格式
            tensor = tensor.squeeze(0)  # 移除通道维度，变为[H, W]
        
        if len(tensor.shape) == 4:
            if tensor.shape[1] ==3:
                tensor = tensor.squeeze(0)
            else:
                tensor = tensor.squeeze(0).squeeze(0)  # 移除通道维度，变为[H, W]
        depth_array = tensor.detach().numpy()
        
        # 使用tifffile保存为TIFF格式
        # TIFF支持直接存储float32数据，无需归一化到0-255
        tifffile.imwrite(output_path, depth_array, photometric='minisblack')
```

**Comment by bell-one** [comment]
If you just use colmap or other software application with only images, scale of scene will not match with lidar metric scale,

How did you get image geometry of the scene?

**Comment by 7yzx** [comment]
> If you just use colmap or other software application with only images, scale of scene will not match with lidar metric scale,
> 
> How did you get image geometry of the scene?

I used a software application made by ourselves like colmap, and I visualize its `.ply` output in meshlab, the metric seems right at  windows($\approx$ 0.7m) and flowers($\approx$ 0.5m)


![Image](https://github.com/user-attachments/assets/69cf91f0-6e1a-4a57-983e-0f7bbcc5cfe2)

And also my gt depth is from iphone lidar. Maybe I need to test another dataset `MuSHRoom` captured using iPhone camera with LiDAR?

**Comment by 7yzx** [comment]
I used a `colmap` dataset that provides several options for the `orientation_method`. When these options are turned off, the depth values match what I need.

```json
        "--orientation-method","none",
        "--center-method", "none",
        "--auto-scale-poses", "False",
        "--assume-colmap-world-coordinate-convention", "False"
```

**Comment by samshott** [comment]
Hello @7yzx, I'm having the same issue, and I see you solved it, can you tell me where you used those options though? If I use it in the ns-process-data command it does not work, it seems like I must pass those directly to Colmap?

**Comment by 7yzx** [comment]
> Hello [@7yzx](https://github.com/7yzx), I'm having the same issue, and I see you solved it, can you tell me where you used those options though? If I use it in the ns-process-data command it does not work, it seems like I must pass those directly to Colmap?你好 

I use it in training, you can use `ns-train splatfacto colmap -h `.
